src/components/constants/button-types.ts

```diff
@@ -0,0 +1,5 @@
+export enum ButtonType {
+  Success = "success",
+  Alert = "alert",
+  Erro = "erro"
```

Comment:
Sempre mantenha a consistência, como o seu código é em inglês, error também precisa ser
```suggestion
  Error = "error"
```

Reply:
commit: 3a149e0753f3f831d576d78d783a134b912e47ef